integration-tests/bats/config.bats

```diff
@@ -208,7 +208,7 @@ teardown() {
     start_sql_server
     
     dolt sql-client --use-db '' -u dolt -P $PORT -q "create database testdb"
-    run dolt sql-client --use-db '' -u dolt -P $PORT -r csv -q "show databases"
+    run dolt sql-client --use-db '' -u dolt -P $PORT --result-format csv -q "show databases"
```

Comment:
Does `-r` not work anymore?

Reply:
`-r` is overloaded now. I think because sql-client inherits args from sql-server as well? Not sure how that happened.

It now means `read-only` or `result-format`.

https://docs.dolthub.com/cli-reference/cli#dolt-sql-client